src/App.js

```diff
@@ -16,31 +19,58 @@ import OldClient from "./apps/VirtualApp/OldClient";
 // import SDIOutApp from "./apps/SDIOutApp/SDIOutApp";
 // import AdminCongress from "./apps/AdminApp/AdminCongress";
 // import AdminStreaming from "./apps/AdminApp/AdminStreaming";
+import { resources } from './translations';
 
-class App extends Component {
+// instance for client side
+i18n
```

Comment:
Can we extract this init into its own file ? 
Let's keep `App.js` as small as possible